docs/src/paradox/getting-started.md

```diff
@@ -1,18 +1,25 @@
-# Command Line Usage
+# Getting Started
+
+No setup is required in order to use to use `scala-native-bindgen`. Statically linked
+executables are provided with each release for Linux and macOS. Head over to the [releases page] to download the latest version for your platform.
+
+ [releases page]: https://github.com/kornilova-l/scala-native-bindgen/releases
+
+## Command Line Usage
```

Comment:
Probably we should mention here that there are two ways to obtain bindgen and provide a link to `using-generated-bindings` page (I will update this page soon)

Reply:
Yes, working on that.

Reply:
All the code snippets on the `using-generated-bindings` page are now tested and/or generated.